Content/Guides/syslog-ng-guide-admin/chapters/syslog-ng-whatsnew.htm

```diff
@@ -6,29 +6,33 @@
-			</li>
-			<li>
-				<p><b>New --remove-orphans option in syslog-ng-ctl stats</b>
-				</p>
-				<p>New option <span class="Code">--remove-orphans</span> has been added to the <span class="Code">stats</span> command.</p>
-			</li>
+		<ul>
 			<li>
-				<p><b>New monitoring() source driver</b>
+				<p><b>New destination: discord()</b>
 				</p>
-				<p>The <span class="Code" oldrole="parameter">monitoring()</span> source allows you to select which statistics of <MadCap:variable name="General.abbrev"></MadCap:variable> you want to monitor. In addition, the statistics are available as structured name-value pairs, so you can format the output similarly to other log messages. That way, you can easily convert the statistics and metrics, for example, into JSON or WELF format. That way, you can send the statistics of your log messages into a monitoring solution<MadCap:conditionaltext MadCap:conditions="General.ose">, for example, into Riemann, Redis, or Graphite</MadCap:conditionaltext>.</p>
-        
+				<p>The <span class="Code">discord()</span> destination driver sends messages to <a href="https://discord.com/">Discord</a> using <a href="https://discord.com/developers/docs/resources/webhook">Discord Webhook</a>.</p>
 			</li>
-            </ul>
+            <li>
+                <p><b>New options for the kafka() destination C implementation</b>
+                </p>
+                <p>Options <span class="Code">batch-lines()</span> and <span class="Code">batch-timeout()</span> have been added.</p>
+            </li>
+            <li>
+                <p><b>Proxy Protocol support</b>
+                </p>
+                <p>Proxy Protocol is now supported in the network() source.</p>
+                <p>If you connect load balancers to your <MadCap:variable name="General.OSE" /> application, <MadCap:variable name="General.OSE" /> identifies every connection that is connected to the load balancers identically by default, regardless of the source IP or the source protocol. Essentially, the load balancer masks the source IP unless you enable <a href="https://www.haproxy.com/blog/haproxy/proxy-protocol/">Proxy Protocol</a> support for your proxy TLS <span class="Code">transport()</span> to inject information about the original connection into the forwarded TCP session.</p>
+            </li>
+            <li>
+                <p><b>New --remove-orphans option in syslog-ng-ctl stats</b>
+                </p>
+                <p>New option <span class="Code">--remove-orphans</span> has been added to the <span class="Code">stats</span> command.</p>
+            </li>
+            <li>
+                <p><b>New monitoring() source driver</b>

```

Comment:
Monitoring() source is a PE-only feature. Please remove it from the OSE admin guide or add a conditional variable to include this only in the PE admin guide.

Reply:
@gaborznagy done, thanks for the comment!